web/tests/test_chapters.py

```
@@ -0,0 +1,634 @@
+        # Create a pending application
+        application = ChapterApplication.objects.create(
+            applicant=self.user,
+            chapter_name="Application Test Chapter",
+            region="Test Region",
+            description="Test description",
+            proposed_activities="Test activities",
+            experience="Test experience",
+        )
+        
+        # Create admin user
+        admin = User.objects.create_user(
```

Ruff (0.8.2)

Local variable `admin` is assigned to but never used

Remove assignment to unused variable `admin`

(F841)